targets.json

```diff
@@ -67,6 +67,61 @@
             }
         }
     },
+    "bayck": {
+        "name": "BAYCK RC",
+        "rx_900": {
+            "nano": {
+                "product_name": "BAYCK 900MHz Nano RX",
+                "lua_name": "BK 900 Nano RX",
+                "layout_file":  "Generic 900.json",
+                "upload_methods": ["uart", "wifi", "betaflight"],
+                "min_version": "3.3.1",
+                "platform": "esp8285",
+                "firmware": "Unified_ESP8285_900_RX"
+            },
+            "nano-pro": {
```

Comment:
Looking at the amp datasheet the LNA has a gain of 16 and we need the below overlay.

"power_lna_gain": 16

Reply:
True, we should put it in, but it's only used for LBT.